src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -299,24 +299,27 @@ export class HuiCardPicker extends LitElement {
       }
     }
 
+    const cardName = customCard
+      ? `${this.hass!.localize(
+          "ui.panel.lovelace.editor.cardpicker.custom_card"
+        )}: ${customCard.name || customCard.type}`
+      : name;
+
     return html`
       <div class="card">
-        <div
+        <button
           class="overlay"
+          type="button"
           @click=${this._cardPicked}
           .config=${cardConfig}
-        ></div>
-        <div class="card-header">
-          ${customCard
-            ? `${this.hass!.localize(
-                "ui.panel.lovelace.editor.cardpicker.custom_card"
-              )}: ${customCard.name || customCard.type}`
-            : name}
-        </div>
+          .title=${cardName}
+        ></button>
+        <div class="card-header">${cardName}</div>
         <div
           class="preview ${classMap({
             description: !element || element.tagName === "HUI-ERROR-CARD",
           })}"
+          inert
```

Comment:
We need to add the [wicg-inert](https://github.com/WICG/inert) polyfill (it's already a sub-dependency from `mwc-dialog` but we need it in our entrypoints).  Latest Firefox still doesn't support it without tweaking config and older browsers certainly don't.

Also should add `aria-hidden="true"` to be safe.

Reply:
I tested the polyfill and I'm unable to make it works on Firefox. Also, the inert polyfill from `mwc-dialog` doesn't work neither.

Reply:
I use Firefox regularly and have never had a problem with the modality of dialogs.  How did you test?

Reply:
Inside combobox, we remove the `inert` property for the overlay (because it's rendered outside the dialog). If I remove this lines of code, combobox is broken in Chrome because we can not click/focus on the overlay. In firefox, the combobox is not broken so I deduce that the inert property is always ignored (even with the polyfill).